lib/code_corps/github/sync/user/record_linker.ex

```diff
@@ -39,18 +38,17 @@ defmodule CodeCorps.GitHub.Sync.User.RecordLinker do
     - If there are multiple matching users, this is an unexpected scenario and
       should error out.
   """
-  @spec link_to(GithubComment.t | GithubIssue.t | GithubPullRequest.t, map) :: linking_result
-  def link_to(%GithubComment{} = comment, %{"comment" => %{"user" => user}}), do: do_link_to(comment, user)
-  def link_to(%GithubIssue{} = issue, %{"issue" => %{"user" => user}}), do: do_link_to(issue, user)
-  def link_to(%GithubPullRequest{} = pull_request, %{"pull_request" => %{"user" => user}}), do: do_link_to(pull_request, user)
+  @spec link_to(GithubComment.t | GithubIssue.t, map) :: linking_result
+  def link_to(%GithubComment{} = comment, %{"user" => user}), do: do_link_to(comment, user)
+  def link_to(%GithubIssue{} = issue, %{"user" => user}), do: do_link_to(issue, user)
```

Comment:
We no longer match the `GithubPullRequest` because the linking to users happens with issues (and tasks). Pull requests can be separate from this affair since the issue users are the same as the task users.